core/src/main/java/com/amplifyframework/predictions/models/LanguageType.java

```diff
@@ -145,11 +145,42 @@
      * @return An enum value of matching language code
      */
     public static LanguageType from(String languageCode) {
-        try {
-            return LanguageType.valueOf(languageCode);
-        } catch (IllegalArgumentException noMatchError) {
-            return UNKNOWN;
+        // Micro-optimization... for fun :)
```

Comment:
Haha :-D Alright, cool.

Also, I did not realize Spanish was #2, above English!

Quizas debo practicarlo con mas frequencia!